custom_components/miio2/manifest.json

```diff
@@ -5,7 +5,7 @@
   "documentation": "https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym",
   "requirements": [
     "construct==2.10.56",
-    "python-miio==0.5.8"
+    "python-miio==0.5.9.1"
```

Comment:
```suggestion
    "python-miio==0.5.9.2"
```

Reply:
because this is the newest version, i suggest to use this, and change PR name accordingly

Reply:
My PR is out of date because @KrzysztofHajdamowicz made another commit here:
https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym/commit/47cbc75b38fb474ad6d36ed20fbb0af516255d53